src/Promotions/InPluginUpsells/Endpoints/ProductRecommendationsRoute.php

```diff
@@ -105,9 +106,13 @@ public function handleRequest(WP_REST_Request $request)
             case 'givewp_reports_recurring_recommendation_dismissed':
                 update_option('givewp_reports_recurring_recommendation_dismissed', time());
                 break;
+
             case 'givewp_reports_fee_recovery_recommendation_dismissed' :
                 update_option('givewp_reports_fee_recovery_recommendation_dismissed', time());
                 break;
+
+            case 'givewp_donors_fee_recovery_recommendation_dismissed':
+                update_option('givewp_donors_fee_recovery_recommendation_dismissed', time());
```

Comment:
Remember to add a break to protect future cases.

Reply:
Resolved: https://github.com/impress-org/givewp/pull/6800/commits/45cbf2edf375b58c1464673744014a757e8166e0